Plugin/CppApi/ArcGISRuntimeToolkit.pri

```diff
@@ -39,7 +39,7 @@ INCLUDEPATH += $$PWD/include
 !android:!win32: PLATFORM_OUTPUT = $$PLATFORM
 android: PLATFORM_OUTPUT = $$PLATFORM/$$ANDROID_ARCH
 win32: {
-  contains(QMAKE_TARGET.arch, x86_64): {
+  contains(QT_ARCH, x86_64): {
```

Comment:
@ldanzinger I found we needed this to build the toolkit to the appropriate `PLATFORM_OUTPUT` folder.